web/components/input/text/TextInput/TextInput.sass

```diff
@@ -0,0 +1,57 @@
+div.text-input
+  margin: 5px
+
+  *
```

Comment:
I'm pretty sure traits defined in `text-input` will be inherited without `*`

Reply:
it doesn't get inherited 😞